netsuite/restlet.py

```diff
@@ -51,7 +51,7 @@ def request(
     def raw_request(
         self,
         script_id: int,
-        payload: Any = None,
+        payload: Any = {},
```

Comment:
Same issue as before.

Reply:
Or shall I adding a type check/value assignment before sending the request instead of changing the default value of the param to a dict?
Thanks,

Reply:
Thought about this a bit... We probably want the client itself to raise some sort of validation error instead of trying to post an empty payload... Since posting an empty payload may result in accidental overwrite. We don't want to enable that.

Reply:
Thanks a lot for your time on this.
I proposed this for 
- some new beginners (like me)
- the payload is not needed
NetSuite returns 400 error causes confusion.

but I totally understand rewrite a param doesn't make sense. Or let me add more comments to the readme about restlet.

Reply:
So what if we do this... why don't we make payload into a required argument? That's still backwards compatible, and you cannot accidentally send up an empty payload.

``` 
def raw_request(
     self,
     script_id: int,
     payload: Any,
      . . .
)

self.raw_request(12, {}) # valid
self.raw_request(12, payload={}) # valid
self.raw_request(12) # TypeError: raw_request() missing 1 required positional argument: 'payload'
```

Reply:
Awesome, I totally agree with this.

Reply:
Okay, if you submit a PR with the changes I will merge it. You don't have to close a PR to update it, btw, you can just keep pushing to your fork.